{
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
        "ID": "CVE-2019-1222222",
        "ASSIGNER": "cve-assign@distributedweaknessfiling.org",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "n/a",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "before 3.1.1"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": {
                    "lang": "eng",
                    "value": "ProblemType"
                }
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "refsource": "CONFIRM",
                "name": "https://www.knot-resolver.cz/2018-01-22-knot-resolver-1.5.2.html",
                "url": "https://www.knot-resolver.cz/2018-01-22-knot-resolver-1.5.2.html"
            }
        ]
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "This CVE Entry does not have a Product. It should return an error; Work Item should not be created."
            }
        ]
    }
}